### PR TITLE
Ets 1394 checkbox style

### PR DESCRIPTION
**Pull request by virgilchiriac**

### CODE REVIEW

**Review by cllagosr — APPROVED** [review]

**Comment by brunogeronimo** [comment]
Since we are creating a new component on this PR (the checkbox component), maybe we should use version `0.20.0` instead of `0.19.2`. What do you think?

File: source/scss/components/_form-checkbox.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.checkmark {
+  border: solid 1px $secondary;
```
Comment (BirgitPohl) [inline source/scss/components/_form-checkbox.scss:2]: Do not use the variable secondary, since it is blue and not gray. Use directly $gray-600

**Review by BirgitPohl — COMMENTED** [review]
Try to use rem a bit more instead of pixels. 16pt is the base-size

File: source/scss/components/_form-checkbox.scss (hunk 1 above)
Comment (BirgitPohl) [reply source/scss/components/_form-checkbox.scss:2]: The intention is to use $secondary

**Review by BirgitPohl — APPROVED** [review]

**Pull request merged**